forum/index.php

```diff
@@ -19,13 +19,19 @@
 
 	More about this license: http://www.question2answer.org/license.php
 */
-
 //	Set base path here so this works with symbolic links for multiple installations
+require_once 'vendor/autoload.php';
 
-	define('QA_BASE_DIR', dirname(empty($_SERVER['SCRIPT_FILENAME']) ? __FILE__ : $_SERVER['SCRIPT_FILENAME']).'/');
+global $client;

```

Comment:
zrobiłem global, bo w tej chwili jest to najwygodniejsze w użyciu - obiecuję to poprawić na jakiegoś (przynajmniej) traita, ale w tej chwili jest to najmniej istotne :)